engine/values/values.go

```diff
@@ -374,6 +378,19 @@ func (v *Values) varEval(varName string) (interface{}, error) {
 	return res.String(), nil
 }
 
+// fromJSON decodes JSON into a structured value, ignoring errors.
+func (v *Values) fromJSON(s string) (interface{}, error) {
+	output, _ := v.mustFromJSON(s)
+	return output, nil
+}
+
+// mustFromJSON decodes JSON into a structured value, returning errors.
+func (v *Values) mustFromJSON(s string) (interface{}, error) {
```

Comment:
How about returning a (possibly zero value) reflect.Value instead of interface?
See `field` implementation, it gets properly considered as a zero value for eg coalesce, default, '<no value>' replacement by the text tempmate engine, etc

Reply:
Good idea. What should we consider as "zero-value" in the context of a JSON-value tho ?
`reflect.ValueOf(bill)` will return the zero `Value`, which is great, but what about empty string, zero numbers, and so on ?